predicators/pretrained_model_interface.py

```diff
@@ -125,6 +127,11 @@ def sample_completions(self,
         cached_prompt, completion_strs = cache_str.split(_CACHE_SEP, 1)
         assert cached_prompt == prompt
         completions = completion_strs.split(_CACHE_SEP)
+
+        logging.debug(f"Querying model {model_id} with new prompt.")
+        # Query the model.
+        completions = self._sample_completions(prompt, imgs, temperature, seed,
+                                               stop_token, num_completions)
```

Comment:
woah woah - what is this? why are you querying the model again?

Reply:
I'd commented out the earlier code that calls this function and caches the result when the caching was throwing an error (because of the foldername being too long), and that's why I had this one here -- and I just forgot to remove this line after I uncommented the earlier code and fixed the caching issue.